lychee-bin/src/options.rs

```diff
@@ -425,6 +445,7 @@ impl Config {
             max_retries: DEFAULT_MAX_RETRIES;
             max_concurrency: DEFAULT_MAX_CONCURRENCY;
             max_cache_age: humantime::parse_duration(DEFAULT_MAX_CACHE_AGE).unwrap();
+            cache_exclude_status: AcceptSelector::new();
```

Comment:
`AcceptSelector` is kinda missnamed now, and something like `StatusCodeSelector` would probably be better. But I didn't want to introduce a breaking change into the lib.

Reply:
Since lychee is not 1.0 yet, it's okay to introduce a breaking change. 
There aren't that many library users that I know of anyway. 
The only one is [lychpy](https://github.com/jb--/lychpy) by @jb--.
(@jb-- fyi, in case you want to update your tool later.)

So, feel free to rename it.